src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Contact.tsx

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import {
+  Fragment,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react'
+
+import { useParams } from 'react-router-dom'
+
+import Edit from './Edit'
+import { StyledDD, StyledEditButton, StyledLink } from './styled'
+import { useFetch } from '../../../../../../../../hooks'
+import configuration from '../../../../../../../../shared/services/configuration/configuration'
+import type { Incident } from '../../../../../../../../types/incident'
+import IncidentDetailContext from '../../../../context'
+import type { Result, SignalReporter } from '../../../../types'
+
+type Props = {
+  incident: Incident
+  showPhone: boolean
+}
+
+export const Contact = ({ incident, showPhone }: Props) => {
+  const [showEdit, setShowEdit] = useState(false)
+  const { post, isSuccess, get, data } = useFetch<Result<SignalReporter>>()
+
+  const { getHistory, getIncident } = useContext(IncidentDetailContext)
+
+  const params = useParams()
+
+  const onClose = useCallback(() => {
+    setShowEdit(false)
+  }, [])
+
+  const submit = useCallback(
+    (data) => {
+      data.sharing_allowed = incident.reporter.sharing_allowed
```

Comment:
What's happening here?

Reply:
the BE needs that value for the post to succeed